src/components/Table.tsx

```diff
@@ -56,12 +67,14 @@ const Table: React.FC<Props> = ({tableData, setTableData, gridTemplateColumns, c
     <div  className={styles.Table}>
       <div style={{
         gridTemplateColumns: gridTemplateColumns,
-        paddingRight: tableIsOverflow ? scrollbarWidth : undefined
+        paddingRight: tableIsOverflow ? scrollbarWidth : undefined,
+        minWidth: responsive ? 'unset' : undefined,
```

Comment:
I think I hardcoded the minwidth previously to 1200 in the css.
We now need this to be more flexible.

I see to possible alternatives for the current solution of making table responsive vs unresponsive:
- pass the minwidth as a parameter to table and tableStateContainer
- make the table+tablestateContainer always width=100% but hardcode the component around it (not sure if this is easy enough in the current setup).

Reply:
Your solution also sounds good but I will keep the responsive parameter for now.